packages/xstate-react/test/useMachine.test.tsx

```diff
@@ -376,14 +491,7 @@ describeEachReactMode('useMachine (%s)', ({ suiteKey, render }) => {
 
     render(<App />);
 
-    expect(rerenders).toBe(
-      suiteKey === 'strict'
-        ? // it's rendered twice for the each state
-          // and the machine gets currently completely restarted in a double-invoked strict effect
-          // so we get a new state from that restarted machine (and thus 2 additional strict renders) and we end up with 4
```

Comment:
Shouldn't this comment be kept to explain why we have 4 rerenders here right now? The usual expectation here is 2 but for the time being we couldn't achieve that ideal result (because we lack rehydratable state)